### PR TITLE
nv21: Add SynthPoRep config

### LINKED ISSUE #666: Add SynthPoRep config to the docs
Add SynthPoRep config and summary of SynthPoRep to the docs

### PR DESCRIPTION
**Pull request by rjan90**
Closes #666 

Add SynthPoRep config and summary of SynthPoRep to the docs

### CODE REVIEW

File: content/en/storage-providers/advanced-configurations/sealing.md [hunk 1]
```diff
@@ -226,6 +213,12 @@ This section controls some of the behavior around sector sealing:
   # type: Duration
   # env var: LOTUS_SEALING_TERMINATEBATCHWAIT
   #TerminateBatchWait = "5m0s"
+
+  # UseSyntheticPoRep, when set to true, will reduce the amount of cache data held on disk after the completion of PreCommit 2 to 11GiB.
```
Comment (TippyFlitsUK) [inline content/en/storage-providers/advanced-configurations/sealing.md:217]:
```suggestion
  # UseSyntheticPoRep, when set to true, will reduce the amount of cache data held on disk after the completion of PreCommit2 to 11GiB.
```

Comment (TippyFlitsUK) [reply content/en/storage-providers/advanced-configurations/sealing.md:217]: Just a suggestion 😉

**Review by TippyFlitsUK — APPROVED** [review]
👍

File: content/en/storage-providers/advanced-configurations/sealing.md (hunk 1 above)
Comment (rjan90) [reply content/en/storage-providers/advanced-configurations/sealing.md]: Thanks! Added

**Pull request merged**